src/ui/js/Overview.js

```diff
@@ -273,6 +274,12 @@ Overview.pageAddGameTable = function(
       gameLinkTd.append($('<a>', {'href': 'game.html?game=' + gameInfo.gameId,
                                   'text': 'Play Game ' + gameInfo.gameId,}));
     } else if (gameType == 'awaitingOpponent') {
+      if (gameInfo.inactivityRaw > 14*24*60*60) {
```

Comment:
Make this a variable at the top of the file --- STALENESS_SECS = 14 * 24 * 60 * 60 or something.  It's a little icky to have a bunch of multiplication inline.  Or if you must do the multiplication inline, at least make STALENESS_DAYS a variable.